internal/server/statvar/statvar_group.go

```diff
@@ -270,6 +276,8 @@ func GetStatVarGroup(
 			},
 		)
 	}
+	// Recount all descendent stat vars after merging
+	adjustDescendentSVCount(result.StatVarGroups, SvgRoot)
```

Comment:
If we're computing  the counts here, is it okay to refer to those above, like:

`result.StatVarGroups[SvgRoot].DescendentStatVarCount += customRootNode.DescendentStatVarCount`

Reply:
good catch, that one is not needed anymore